chrome/content/zotero/xpcom/itemTreeView.js

```diff
@@ -260,6 +261,44 @@ Zotero.ItemTreeView.prototype.setTree = Zotero.Promise.coroutine(function* (tree
 });
 
 
+Zotero.ItemTreeView.prototype.setSortColumn = function() {
+	var colId, col, dir, currentCol, currentDir;
```

Comment:
Which is why I think just sticking to either only `let` or only `var` for new code would be the best way to go about things. Really, there isn't one best way to do this (different approaches have different drawbacks and benefits). Mixing the 2 leads to fairly fruitless discussions about style, when in reality we are just trying to write code that has best readability and is consistent with itself.